src/models/kernels.hh

```diff
@@ -80,13 +80,11 @@ namespace rascal {
         size_t ii_A{0};
         for (auto & manager_a : managers_a) {
           size_t ii_B{0};
-          auto && propA{
-              manager_a->template get_validated_property_ref<Property_t>(
-                  representation_name)};
+          auto && propA{manager_a->template get_property_ref<Property_t>(
+              representation_name, true, true)};
```

Comment:
There are a few places where you force_creation and you shouldn't. here for instance we build a kernel and if one forgot to compute the representation I think it is more informative to fail making sure the user is not surprised to get a kernel filled with zeros.

Reply:
I changed the ones in kernels to not forced, but in the representation calculator all are still forced, is this okay like this?